articles/azure-monitor/app/tutorial-custom-metrics.md

```diff
@@ -0,0 +1,370 @@
+
+Trend tracking in a metric every second, or at an even more granular interval can result in:
+
+- Increased data storage costs. There's a cost associated with how much data you send to Azure Monitor. (The more data you send the greater the overall cost of monitoring.)
+- Increased network traffic/performance overhead. (In some scenarios this overhead could have both a monetary and application performance cost.)
+- Risk of ingestion throttling. (The Azure Monitor service drops ("throttles") data points when your app sends a high rate of telemetry in a short time interval.)
+
+Throttling is a concern as it can lead to missed alerts. The condition to trigger an alert could occur locally and then be dropped at the ingestion endpoint due to too much data being sent. We don't recommend using `TrackMetric()` for .NET and .NET Core unless you've implemented your own local aggregation logic. If you're trying to track every instance an event occurs over a given time period, you may find that [`TrackEvent()`](./api-custom-events-metrics.md#trackevent) is a better fit. Though keep in mind that unlike custom metrics, custom events are subject to sampling. You can still use `TrackMetric()` even without writing your own local pre-aggregation, but if you do so be aware of the pitfalls.
+
+In summary `GetMetric()` is the recommended approach since it does pre-aggregation, it accumulates values from all the Track() calls and sends a summary/aggregate once every minute. `GetMetric()` can significantly reduce the cost and performance overhead by sending fewer data points, while still collecting all relevant information.
+
+## Getting a TelemetryClient instance
+
+Get an instance of `TelemetryClient` from the dependency injection container in **HomeController.cs**:
+
+```csharp
+//... additional code removed for brevity
+using Microsoft.ApplicationInsights;
+
+namespace AzureCafe.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly ILogger<HomeController> _logger;
+        private AzureCafeContext _cafeContext;
+        private BlobContainerClient _blobContainerClient;
+        private TextAnalyticsClient _textAnalyticsClient;
+        private TelemetryClient _telemetryClient;
+
+        public HomeController(ILogger<HomeController> logger, AzureCafeContext context, BlobContainerClient blobContainerClient, TextAnalyticsClient textAnalyticsClient, TelemetryClient telemetryClient)
+        {
+            _logger = logger;
+            _cafeContext = context;
+            _blobContainerClient = blobContainerClient;
+            _textAnalyticsClient = textAnalyticsClient;
+            _telemetryClient = telemetryClient;
+        }
+
+        //... additional code removed for brevity
+    }
+}
+```
+
+`TelemetryClient` is thread safe.
+
+## TrackMetric
+
+> [!NOTE]
```

Comment:
this information is already covered earlier in this doc, so can be omitted from here to avoid duplication.

Reply:
Fixed.